### DISCUSSION TITLE
How fast is the TEXT generation for you? I am getting 0.5 to 1s a line of text? That's SLOW

### DISCUSSION THREAD

**Discussion by GithubStable** [lllyasviel/Omost#51] [General]
I have these logs:
> 
> Load to GPU: CLIPTextModel
> Load to GPU: CLIPTextModel
> Unload to CPU: CLIPTextModel
> Unload to CPU: CLIPTextModel
> Load to GPU: UNet2DConditionModel
> 100%|██████████████████████████████████████████████████████████████████████████████████| 25/25 [00:16<00:00,  1.56it/s]
> Unload to CPU: UNet2DConditionModel
> Load to GPU: AutoencoderKL
> Unload to CPU: AutoencoderKL
> You shouldn't move a model that is dispatched using accelerate hooks.
> Load to GPU: LlamaForCausalLM
> The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
> Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.

Wonder if I have any problem, or is it supposed to be THAT slow? 
Please tell me how fast the prompt generations are for you please, 
For me it's like Half to One second per line of text, and each prompt has like 20-40 lines, that's quite slow to generate a prompt, am I missing something?
